app.js

```diff
@@ -1,8 +1,154 @@
-// import functions and grab DOM elements
+import { renderGoblin } from './utils.js';
 
-// let state
+const form = document.getElementById('form');
+const goblinList = document.getElementById('goblins');
+const heroHPel = document.getElementById('heroHP');
+const defeatedGoblinsEl = document.getElementById('defeatedGoblins');
 
-// set event listeners 
-  // get user input
-  // use user input to update state 
-  // update DOM to reflect the new state
+
+function playCheer() {
+    new Audio(`assets/18364__jasinski__bb-claprhm.mp3`).play();
+}
+function playHit() {
+    new Audio(`assets/214989__amishrob__baseballhitandcrowdcheer.mp3`).play();
+}
+function playStrike() {
+    new Audio(`assets/45110__sanus-excipio__thats-it.wav`).play();
+}
+
+
+let defeatedGoblins = [];
+let currentID = 6;
+let goblins = [{
+    name: `Randy Johnson`, id: 1, hp: 8
+}, {
+    name: `Terry Pendleton`, id: 2, hp: 7
+}, {
+    name: `Greg Maddux`, id: 3, hp: 5
+}, {
+    name: `Roger Clemens`, id: 4, hp: 6
+}, {
+    name: `Mario Soto`, id: 5, hp: 5
+}];
+let heroHP = 3;
+
+
+form.addEventListener('submit', (e) => {
+    console.log('click');
+    e.preventDefault();
+    const data = new FormData(form);
+
+    const goblinName = data.get('goblin-name');
+
+    const newGoblin = {
+        id: currentID,
+        name: goblinName,
+        hp: Math.ceil(Math.random() * 5),
```

Comment:
This means you don't have a default hp for new goblins. I'll give it to you, but the rubric does call for a default hp to be used.